src/dlr.cc

```diff
@@ -243,23 +234,23 @@ extern "C" int CreateDLRModel(DLRModelHandle* handle, const char* model_path, in
   ctx.device_id = dev_id;
 
   std::vector<std::string> path_vec = MakePathVec(model_path);
+  std::vector<std::string> files = FindFiles(path_vec);
 
-  DLRBackend backend = dlr::GetBackend(path_vec);
+  DLRBackend backend = dlr::GetBackend(files);
   DLRModel* model;
   try {
     if (backend == DLRBackend::kTVM) {
```

Comment:
Could you replace this block with CreateDLRModelPtr()? Sorry we should've done that earlier

Reply:
I dont think its that simple. CreateDLRModelPtr allocates on the stack, but CreateDLRModel allocates with new